book/src/user/startup.md

```diff
@@ -0,0 +1,190 @@
+- full verification: check signatures, proofs, spent transparent outputs, and all the other consensus rules
+
+Mempool transactions are always fully verified.
+
+Zebra also starts ongoing tasks to batch verify signatures and proofs.
+
+```
+zebrad::commands::start: initializing verifiers
+init{config=Config { debug_skip_parameter_preload: false, ... } ... }: zebra_consensus::primitives::groth16::params: checking and loading Zcash Sapling and Sprout parameters
+init{config=Config { checkpoint_sync: true, ... } ... }: zebra_consensus::chain: initializing chain verifier tip=None max_checkpoint_height=Height(1644839)
+...
+init{config=Config { debug_skip_parameter_preload: false, ... } ... }: zebra_consensus::chain: Groth16 pre-download and check task finished
+```
+
+### Initialize Transaction Mempool
+
+Zebra syncs transactions from other peers, verifies them, then gossips their hashes.
+
+This involves a number of ongoing tasks.
+
+The mempool isn't activated until Zebra reaches the network consensus chain tip.
+
+```
+zebrad::commands::start: initializing mempool
+zebrad::components::mempool::crawler: initializing mempool crawler task
+zebrad::components::mempool::queue_checker: initializing mempool queue checker task
+zebrad::components::mempool::gossip: initializing transaction gossip task
+```
+
+### Syncing the Block Chain
+
+#### Initialize Block Syncer
+
+Zebra syncs blocks from other peers, verifies them, then gossips their hashes:
+1. Download the genesis block by hash
+2. Ask a few peers for the hashes of the next 500 blocks
+3. Download the blocks based on the hashes from peers
+4. Verify, save, and gossip the downloaded blocks
+5. Repeat from step 2 until we run out of blocks to sync
+
+This involves a number of ongoing tasks.
+
+```
+zebrad::commands::start: initializing syncer
+zebrad::components::sync::gossip: initializing block gossip task
+```
+
+#### Sync to the Chain Tip
+
+Starting at the cached state chain tip, Zebra syncs to the network consensus chain tip, using ongoing tasks.
+
+Zebra also has an ongoing sync progress task, which logs progress towards the tip every minute.
+
+```
+zebrad::commands::start: spawned initial Zebra tasks
+zebrad::components::sync: starting genesis block download and verify
+zebrad::commands::start: initial sync is waiting to download the genesis block sync_percent=0.000 % current_height=None
+sync:checkpoint: zebra_consensus::checkpoint: verified checkpoint range block_count=1 current_range=(Unbounded, Included(Height(0)))
+sync:try_to_sync: zebrad::components::sync: starting sync, obtaining new tips state_tip=Some(Height(0))
+sync:try_to_sync: zebrad::components::sync: extending tips tips.len=1 in_flight=499 lookahead_limit=2000 state_tip=Some(Height(0))
+sync:try_to_sync:obtain_tips:checkpoint: zebra_consensus::checkpoint: verified checkpoint range block_count=400 current_range=(Excluded(Height(0)), Included(Height(400)))
```

Comment:
I think we can add `...` here and then show a line that prints more than `0.000%` to make the progress more clear.

Reply:
Thanks, that's a good idea:

```suggestion
sync:try_to_sync:obtain_tips:checkpoint: zebra_consensus::checkpoint: verified checkpoint range block_count=400 current_range=(Excluded(Height(0)), Included(Height(400)))
...
zebrad::commands::start: estimated progress to chain tip sync_percent=0.537 % current_height=Height(9119) remaining_sync_blocks=1687657 time_since_last_state_block=PT0S
...
```